aspnetcore/test/http-files.md

```diff
@@ -146,13 +440,12 @@ The Visual Studio 2022 `.http` file editor doesn't have all the features that th
 * Save response body to file
 * Certificate based authentication
 * Prompt variables
-* System variables
 * Customize response preview
 * Per-request settings
 
 ## Use the `.http` file editor
 
-Create an `.http` file by using the **Add New Item** dialog or by renaming the extension of any text file to `.http`. Send a request by selecting the green "run" button to the left of the request. The response appears in a separate pane.
+Create an `.http` file by using the **Add New Item** dialog or by renaming the extension of any text file to `.http`. Send a request by selecting the `Send Request` or `Debug` link above the line that creates a request. The response appears in a separate pane.
```

Comment:
I think we can remove this paragraph. It's no longer accurate because we added an HTTP item template, and you covered that in the following new section.